src/mpi/init/finalize.c

```diff
@@ -155,11 +155,6 @@ int MPI_Finalize(void)
 
     MPIR_ERRTEST_INITIALIZED_ORDIE();
 
-#ifdef HAVE_HWLOC
-    hwloc_topology_destroy(MPIR_Process.hwloc_topology);
-    hwloc_bitmap_free(MPIR_Process.bindset);
-#endif
-
 #ifdef HAVE_NETLOC
     switch (MPIR_Process.network_attr.type) {
         case MPIR_NETLOC_NETWORK_TYPE__TORUS:
```

Comment:
Could you move the `NETLOC` block along with the `HWLOC` block? They are logically in the same category, right?

Reply:
Yeah. You are right. I fixed it.